fastlane/Fastfile

```diff
@@ -41,4 +42,18 @@ platform :android do
   end
 end
 
-# how to used it fastlane android alpha storepass:'#KEYSTORE_PASSWORD' keypass:'#ALIAS_PASSWORD'
\ No newline at end of file
+after_all do |lane|
+    telegram(
+        url_horn: "$TELEGRAM_WEBHOOKS",
+        message: "Fastlane is OK"
```

Comment:
Yes, we have more information inside the telegram.rb on fastlane action folder but this is the first version 👍